packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -752,4 +752,16 @@
     <label>Résumé</label>
     <name>quantic_Summary</name>
   </customLabels>
+  <customLabels>
+    <label>La réponse générée est cachée</label>
+    <name>quantic_GeneratedAnswerIsHidden</name>
+</customLabels>
+<customLabels>
+  <label>Réponse générée: {{0}}</label>
+  <name>quantic_AnswerGenerated</name>
+</customLabels>
+<customLabels>
+  <label>Génération de la réponse</label>
```

Comment:
```suggestion
  <label>En attente de la réponse</label>
```

Maybe?

Reply:
the screen reader announces this label while the generating answer is streaming, so yes we are waiting for the answer but more precisely the answer is being generated.